src/components/DeckMechanisms/index.js

```diff
@@ -351,6 +364,43 @@ export default class DeckMechanisms extends React.Component {
       deck: this.getIncreasedDeckWeight({ state, reset }),
     }))
 
+  resolveManaRNG = state => {
+    switch (this.state.RNG) {
+      case 'UNFRIENDLY': {
+        state.specifics.activeFrozenCores = 0
+        state.specifics.activeDawnsparks = 0
+        break
+      }
+
+      case 'REGULAR': {
+        const { activeFrozenCores, activeDawnsparks } = state.specifics
+
+        state.specifics.activeFrozenCores = getBinomialRandomVariableResult(
+          activeFrozenCores,
+          FROZEN_CORE_STAYS
+        )
+        state.specifics.activeDawnsparks = getBinomialRandomVariableResult(
+          activeDawnsparks,
+          DAWNSPARKS_STAYS
+        )
+        break
+      }
+
+      default:
```

Comment:
Maybe this for clarity?
```suggestion
      case 'FRIENDLY':
      default:
```

Reply:
It stands for FRIENDLY and UNFRIENDLY - should I add both?